src/Common/Core/Twig/Extensions/BaseTwigModifiers.php

```diff
@@ -37,7 +37,24 @@ public static function formatCurrency($string, $currency = 'EUR', $decimals = nu
             default:
         }
 
-        return $currency.'&nbsp;'.number_format((float) $string, $decimals, ',', '&nbsp;');
+        return $currency.'&nbsp;'.static::formatNumber($string, $decimals);
```

Comment:
When using php-cs-fixer --level=psr2, spaces are used.
I prefer spaces, since everywhere in Fork we use spaces.